virttest/qemu_vm.py

```diff
@@ -1192,12 +1192,15 @@ def add_boot(devices, boot_order, boot_once, boot_menu, boot_strict):
                 logging.warn("-boot on s390x only support boot strict=on")
                 return "-boot strict=on"
             cmd = " -boot"
-            patterns = ["order", "once", "menu", "strict"]
+            patterns = ["order", "once", "menu", "strict", "reboot-timeout"]
+            boot_reboot_timeout = params.get('boot_reboot-timeout', 60)
```

Comment:
It is an optional parameter for the test, so should not have the default value. That means, it would be appended to the command line only if user set its value.

Reply:
updated